content/version/1/4/code-of-conduct.md

```diff
@@ -28,34 +28,35 @@ include:
 Examples of unacceptable behavior by participants include:
 
 * The use of sexualized language or imagery and unwelcome sexual attention or
-  advances
+  advances after being advised of its unwelcomeness
 * Trolling, insulting/derogatory comments, and personal or political attacks
-* Public or private harassment
+* Repeated public or private harassment
```

Comment:
Again, bizarre, same issue.

Reply:
The word "repeated" makes zero change to this clause.  At all.  The word "harassment" by definition refers to a repeated behavior.  Look it up.

Your comment only proves the necessity for this clarification.

Reply:
It actually does not: https://www.google.com/search?q=define%3Aharassment

Reply:
According to the [U.S. Equal Employment Opportunity Commission](https://www.eeoc.gov/laws/types/harassment.cfm):

>Petty slights, annoyances, and isolated incidents (unless extremely serious) will not rise to the level of illegality. To be unlawful, the conduct must create a work environment that would be intimidating, hostile, or offensive to reasonable people.

From a [legal dictionary](https://legal-dictionary.thefreedictionary.com/harassment) (emphasis added):

> harassment: the act of *systematic and/or continued* unwanted and annoying actions of one party or a group, including threats and demands. The purposes may vary, including racial prejudice, personal malice, an attempt to *force someone to quit a job* or grant sexual favors, apply illegal pressure to collect a bill, or merely gain sadistic pleasure from making someone fearful or anxious.

From dictionary.com:

>harassment: the act or an instance of harassing, or disturbing, pestering, or troubling repeatedly; persecution

WordNet:

>the act of tormenting by continued persistent attacks and criticism

Wiktionary:

>Persistent attacks and criticism causing worry and distress.

Reply:
The fact is, almost all dictionaries have "repeated" or a similar word in the definition.  It's good to clarify things, otherwise the document is ambiguous.  See [my comment here](https://github.com/ContributorCovenant/contributor_covenant/pull/580#discussion_r219928217) for why it's a good idea to not accuse people of things after one isolated incident.

Reply:
It's harassment if you talk to someone and use a slur. You don't have to repeat the slur to suddenly meet the bar for harassment.

Reply:
Who or what defines a slur?